src/DTMFSender.ts

```diff
@@ -0,0 +1,113 @@
+class DTMFSender {
+  private _outputNode: MediaStreamAudioDestinationNode;
+  private _outputStream: MediaStream;
```

Comment:
I typically don't use the `_` on private fields or functions in TypeScript because the compiler will prevent you from calling them from elsewhere.